pkg/kubectl/cmd/edit.go

```diff
@@ -184,7 +194,7 @@ func RunEdit(f *cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 	}
 	objs, err := resource.AsVersionedObjects(infos, defaultVersion.String(), encoder)
 	if err != nil {
-		return err
+		allErrs = append(allErrs, err)
```

Comment:
I think you're right - I don't believe it is safe to continue. Can someone else verify this?